packages/core/src/scripts/cli.ts

```diff
@@ -85,7 +85,9 @@ export async function cli(cwd: string, argv: string[]) {
   }
 
   if (command === 'build') {
-    return build(cwd, defaultFlags(flags, { frozen: false, prisma: true, ui: true }));
+    return build(cwd, defaultFlags(flags, { frozen: false, prisma: true, ui: true })).then(() =>
+      process.exit(0)
+    );
```

Comment:
The `cli` function shouldn't call `process.exit(0)`, as that will impact on downstream users of the `cli` export.
This needs to be in `scripts/index.ts`.

I have fixed this in fca0f5ce9 as part of https://github.com/keystonejs/keystone/pull/8790 as it was causing my tests to fail.